### PR TITLE
Simplify adding a new connector to integration tests.

### LINKED ISSUE #1264: Generalize integration test's physical.mongodb.filesystems to work with new connectors
Current instructions led to a integration test hang once the new connector was enabled. Once an interpreter and new external filesystem type was added to TestConfig the tests started running and failing as expected. I ran into this while adding a PostgreSQL "skeleton" and will be needed for other connectors as well.

### PR DESCRIPTION
**Pull request by drostron**
There is just a simple pass to make it easier for a connector implementor to add it to integration tests _(just a few lines)_.

Resolves #1264  

NB: Once merged the associated wiki will require an update.

### CODE REVIEW

File: it/src/test/scala/quasar/TestConfig.scala [hunk 1]
```diff
@@ -74,10 +74,10 @@ object TestConfig {
       envName: String,
       p: ADir
     ): OptionT[Task, Task[(S ~> Task, Task[Unit])]] =
-      TestConfig.loadConfig(envName) map (c =>
-        pf.lift((c, p)) getOrElse Task.fail(new RuntimeException(
-          s"Unsupported filesystem config: $c"
-        )))
+      TestConfig.loadConfig(envName) flatMapF (c =>
```
Comment (drostron) [inline it/src/test/scala/quasar/TestConfig.scala]: Addresses the hang when a filesystem isn't supported — an error is now emitted.

**Comment by jedesah** [comment]
LGTM. I think you are missing a `revision.md` file, though. 😄

**Comment by drostron** [comment]
Ah, indeed. Will add one.

**Comment by drostron** [comment]
Relatedly, https://github.com/slamdata/devtools/pull/12 is still open.

**Pull request closed without merging**